### PR TITLE
Install tensorflow-docs requirements for notebook check

### PR DESCRIPTION
**Pull request by mpharrigan**
The check seemed to be failing on another PR. This fixes it. cc @rmlarose who wrote the original downloader script.

### CODE REVIEW

**Comment by mpharrigan** [comment]
This fixes the issue seen in https://github.com/quantumlib/ReCirq/pull/184 and the issue seen in https://github.com/quantumlib/ReCirq/pull/184 is fixed by this

**Comment by rmlarose** [comment]
Thanks! check/nbformat used to do the pip install - I missed that it was removed in https://github.com/quantumlib/Cirq/pull/4131 ([diff](https://github.com/quantumlib/Cirq/pull/4131/files#diff-3bccb333c0101f89c48efae49d0b18316d17cd6983a72f2e6bcb310e2f574d43L31-L38)).

File: dev_tools/nbfmt [hunk 1]
```diff
@@ -24,6 +24,12 @@ if [[ ! -f dev_tools/nbformat ]]; then
   chmod u+x dev_tools/nbformat
 fi
 
+# Check if cirq/check/nbformat exists, if not grab it.
```
Comment (rmlarose) [inline dev_tools/nbfmt:27]: Update comment

Comment (mpharrigan) [reply dev_tools/nbfmt:27]: Thanks

**Comment by mpharrigan** [comment]
I'm going to force merge because the build failure is unrelated and fixed by #184. I'd rather keep the PRs separate.

**Pull request merged**